### PR TITLE
fix: Add GitHub app URL to env var

### PR DESCRIPTION
**Pull request by novakzaballa**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have run [`pre-commit`](https://docs.flagsmith.com/platform/contributing#pre-commit) to check linting
- [x] I have added information to `docs/` if required so people know about the feature!
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [ ] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes

- Create the new environment variable GITHUB_APP_URL to install the Flagsmith GitHub App.

- Correct the capitalization of an occurrence of the GitHub word
## How did you test this code?

_Please describe._

### CODE REVIEW

**Comment by matthewelwell** [comment]
@novakzaballa feels like there's too many failures in the E2E tests for them to just be flaky?

**Comment by matthewelwell** [comment]
> @novakzaballa feels like there's too many failures in the E2E tests for them to just be flaky?

Hmm... weirdly they all passed on re-running but there were 5 failures originally!

File: frontend/api/index.js [hunk 1]
```diff
@@ -113,13 +113,17 @@ app.get('/config/project-overrides', (req, res) => {
       value: envToBool('DISABLE_INVITE_LINKS', false),
     },
     { name: 'albacross', value: process.env.ALBACROSS_CLIENT_ID },
-    {name: 'useSecureCookies', value: envToBool('USE_SECURE_COOKIES', true)},
-    {name: 'cookieSameSite', value: process.env.USE_SECURE_COOKIES}
+    { name: 'useSecureCookies', value: envToBool('USE_SECURE_COOKIES', true) },
+    { name: 'cookieSameSite', value: process.env.USE_SECURE_COOKIES },
+    {
+      name: 'githubApURL',
```
Comment (matthewelwell) [inline frontend/api/index.js]:
```suggestion
      name: 'githubAppURL',
```

File: frontend/web/components/IntegrationList.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ class Integration extends Component {
 
   openChildWin = () => {
     const childWindow = window.open(
-      'https://github.com/apps/flagsmith-github-integration/installations/select_target',
+      `${Project.githubApURL}`,
```
Comment (matthewelwell) [inline frontend/web/components/IntegrationList.js]:
```suggestion
      `${Project.githubAppURL}`,
```

**Review by matthewelwell — CHANGES_REQUESTED** [review]

File: frontend/api/index.js (hunk 1 above)
Comment (kyle-ssg) [reply frontend/api/index.js]: Am I right in saying there's a Python equivalent of this file which needs to be updated @matthewelwell ?

Comment (matthewelwell) [reply frontend/api/index.js]: Yes, good point. 

`api/app/settings/common.py` and `api/app/views.py` will both need updating too.

Comment (kyle-ssg) [reply frontend/api/index.js]: Added

**Review by matthewelwell — APPROVED** [review]

**Review by kyle-ssg — APPROVED** [review]

**Review by novakzaballa — COMMENTED** [review]
LGTM

**Pull request merged**